pandas/tests/indexes/multi/test_reindex.py

```diff
@@ -101,3 +101,13 @@ def test_reindex_non_unique():
     msg = "cannot handle a non-unique multi-index!"
     with pytest.raises(ValueError, match=msg):
         a.reindex(new_idx)
+
+
+def test_reindex_fill_added_rows():
+    # GH: 23693
+    i = pd.MultiIndex.from_tuples([("a", "b"), ("d", "e")])
```

Comment:
can you use MultiIndex with out the "pd"

Reply:
and pls avoid 1-letter variable names.  `mi` is a good choice for MultiIndex obj

Reply:
Done